apps/reducers/lib/mix/tasks/etl.stores.ex

```diff
@@ -11,7 +11,7 @@ defmodule Mix.Tasks.Etl.Stores do
     DB.Reducer.State.key(e_ctx, "stats")
   end
 
-  def run(:delete) do
+  def run(["delete"]) do
```

Comment:
Could we parse the command line args and use that to determine which run function to call?  We could also move the IO.puts with the mix env to just the main run function.